src/components/dialogBox.jsx

```diff
@@ -0,0 +1,204 @@
+import React from "react";
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import CodeIcon from '@material-ui/icons/Code';
+import CloseIcon from '@material-ui/icons/Close';
+import TextField from '@material-ui/core/TextField';
+import GetAppIcon from '@material-ui/icons/GetApp';
+import FilterNoneOutlinedIcon from '@material-ui/icons/FilterNoneOutlined';
+import useClipboard from "react-use-clipboard";
+import { handleShareToTwitterClick } from "../utils";
+import { Snackbar } from "@material-ui/core";
+import Alert from "@material-ui/lab/Alert";
+import DialogContentText from '@material-ui/core/DialogContentText';
+
+export const DialogBox = ({ repoList = ["apache/apisix"],}) => {
+  
+  const title = window.location.href.includes("contributorOverTime")
+    ? "contributor over time"
+    : "monthly active contributor"
+    console.log('shareText:',title)
+  const Url = window.location.href.split('repo=apache/apisix')
+  const Tail = Url[1]
+  const OverTimegetShareParams = () => `?chart=contributorOverTime&repo=${repoList.join(",")}${Tail}`
+  const MonthlyActivegetShareParams = () => `?chart=contributorMonthlyActivity&repo=${repoList.join(",")}${Tail}`;
+  const OverTimevalue = `<iframe style={{ width: "100%", height: "auto" , minWidth: "600px", minHeight: "1000px" }} src="https://git-contributor.com/${OverTimegetShareParams()}" frameBorder="0"></iframe>`
+  const MonthlyActivevalue = `<iframe style={{ width: "100%", height: "auto" , minWidth: "600px", minHeight: "1000px" }} src="https://git-contributor.com/${MonthlyActivegetShareParams()}" frameBorder="0"></iframe>`
```

Comment:
We generally use small-hump nomenclature to declare variables, and large-hump nomenclature looks more like a component